src/runtime-rs/crates/resource/src/share_fs/utils.rs

```diff
@@ -53,9 +53,9 @@ pub(crate) fn get_host_rw_shared_path(id: &str) -> PathBuf {
 fn do_get_guest_any_path(target: &str, cid: &str, is_volume: bool, is_virtiofs: bool) -> String {
     let dir = PASSTHROUGH_FS_DIR;
     let guest_share_dir = if is_virtiofs {
-        Path::new("/")
+        Path::new("/").to_path_buf()
```

Comment:
I don't understand why we need the to_path_buf()

Reply:
Thanks for your review, has removed the `to_path_buf`.